packages/geo/src/lib/search/search-results/search-results-add-button.component.ts

```diff
@@ -97,4 +113,18 @@ export class SearchResultAddButtonComponent implements OnInit {
     this.map.removeLayer(oLayer);
   }
 
+  isInResolutionsRange(resolution: number) {
+    const minResolution = this.layer.data.minResolution;
+    const maxResolution = this.layer.data.maxResolution;
+    this.inRange$.next(resolution >= minResolution && resolution <= maxResolution);
+    this.tooltip = this.computeTooltip();
```

Comment:
this.computeTooltip should not be in a method named isInResolutionRange.

Reply:
fixed